cves/kernel/CVE-2021-29657.yml

```diff
@@ -448,7 +460,9 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: The mistake was not disabling the nested virtualization feature. This would be 
```

Comment:
Any insight into why the feature wasn't tested or heavily used or why it's not default disable